data/amd/register/msr/MSRC000_00E8.yml

```diff
@@ -0,0 +1,30 @@
+- name: MSRC000_00E8
+  long_name: "Read-Only Max Performance Frequency Clock Count"
```

Comment:
This should be `long_name: "Read-Only Actual Performance Frequency Clock Count"`